master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -78,7 +78,10 @@ spec:
 | usageReportingEnabled              | Reports anonymous {{site.prodname}} version number and cluster size to projectcalico.org. Logs warnings returned by the usage server. For example, if a significant security vulnerability has been discovered in the version of {{site.prodname}} being used. | boolean | boolean | `true` |
 | usageReportingInitialDelaySecs     | Minimum initial delay before first usage report, in seconds. | int | int | `300` |
 | usageReportingIntervalSecs         | The interval at which Felix does usage reports, in seconds.  The default is 1 day.  | int | int | `86400` |
-
+| vxlanEnabled                       | Automatically set when needed: whether Felix should create the VXLAN tunnel device for VXLAN networking. | boolean | boolean | `false` |
```

Comment:
Needs to be clearer that this should not be used by users.  

(I thought, in error, that this was a way of disabling VXLAN use by calico to make it drop back to no encapsulation on pools).

Reply:
Or not documented?

Reply:
Thanks, reworded.